tests/items/item_test.py

```diff
@@ -79,15 +85,110 @@ def test_get_children(cobbler_api):
     assert result == []
 
 
-def test_descendants(cobbler_api):
-    # Arrange
-    titem = Item(cobbler_api)
+def compare_sets(set1, set2):
+    return set(set1) == set(set2) & set(set1) and set(set2) == set(set2) & set(set1)
```

Comment:
Isn't this the same as `set(set1) == set(set2)`?

Reply:
Yes! certainly. Thank you! I will fix this.

Reply:
Fixed